javascript/javascript1/week3/noNoNoNoYes.js

```diff
@@ -0,0 +1,35 @@
+const notes = [];
+
+function saveNote(content, id) {
+  notes.push({ content, id });
+}
+
+function getNote(id) {
+  if (typeof id !== "number") {
+    console.log("Error: ID must be a number.");
```

Comment:
Nice and concise implementation! ⭐  Very nice to sanity check the input here! Perhaps the same should be done when saving notes? 😄 

The task also asks you to introduce a new feature which seems to be missing. 🥺